docs/README.md

```diff
@@ -4,7 +4,7 @@ This is the main Iroha 2 documentation that you will find useful:
 
 - [Tutorial](https://hyperledger.github.io/iroha-2-docs/)
 - [API Reference](https://hyperledger.github.io/iroha-2-docs/api/torii-endpoints)
-- [Configuration Reference](./source/references/config.md)
+<!-- - [Configuration Reference](./source/references/config.md) -->
```

Comment:
why leave as a comment?

Reply:
Because it will be back when https://github.com/hyperledger/iroha-2-docs/pull/397 is merged.